docs/build/puppeteer-scraper-tutorial.md

```diff
@@ -339,11 +390,14 @@ const buttonSelector = 'div.show-more > button';
 while (true) {
     log.info('Waiting for the "Show more" button.');
     try {
-        await page.waitFor(buttonSelector, { timeout }); // Default timeout first time.
-        timeout = 2000; // 2 sec timeout after the first.
+        // Default timeout first time.
+        await page.waitFor(buttonSelector, { timeout });
+        // 2 sec timeout after the first.
+        timeout = 2000;
     } catch (err) {
         // Ignore the timeout error.
-        log.info('Could not find the "Show more button", we\'ve reached the end.');
+        log.info(`Could not find the "Show more button",
+            we\'ve reached the end.`);
```

Comment:
Watch out here. The linebreak and whitespace will all be included in the message. Better split it with `+` into two strings if you want to place it on two lines.